llvm/lib/Transforms/Utils/BasicBlockUtils.cpp

```diff
@@ -333,6 +333,10 @@ bool llvm::MergeBlockIntoPredecessor(BasicBlock *BB, DomTreeUpdater *DTU,
   // Finally, erase the old block and update dominator info.
   DeleteDeadBlock(BB, DTU);
 
+  // Remove redundant "llvm.dbg" instrunctions after blocks merged.
```

Comment:
nit: after blocks have been merged

Reply:
Thanks, the advice is acceptable, and the comment has been modified.